spec/system/publishers/publishers_can_select_a_job_application_for_interview_spec.rb

```diff
@@ -244,27 +248,56 @@
           expect(publisher_ats_reference_request_page.timeline_titles.map(&:text)).to eq(["Marked as complete", "Marked as interviewing"])
         end
 
-        scenario "changing our mind and using TV after all" do
-          expect(publisher_ats_reference_request_page).to be_displayed
-          publisher_ats_reference_request_page.use_tv_anyway_link.click
+        context "when changing our mind and using TV after all" do
+          before do
+            publisher_ats_reference_request_page.use_tv_anyway_link.click
+          end
 
-          expect(publisher_ats_collect_references_page).to be_displayed
-          choose "Yes"
-          click_on "Save and continue"
+          context "without collecting references" do
+            before do
+              choose "No"
+              click_on "Save and continue"
+            end
 
-          expect(publisher_ats_ask_references_email_page).to be_displayed
-          # don't contact applicant
-          choose "No"
-          click_on "Save and continue"
+            it "stops immediately" do
```

Comment:
I think we could describe this test better than "stops immediately". Maybe it "redirects to the pre interview checks page"?